src/source/tile.js

```diff
@@ -74,12 +75,17 @@ class Tile {
     workerID: number | void;
     vtLayers: {[string]: VectorTileLayer};
     mask: Mask;
+
+    neighboringTiles: ?Object;
+    dem: ?DEMData;
     aborted: ?boolean;
     maskedBoundsBuffer: ?VertexBuffer;
     maskedIndexBuffer: ?IndexBuffer;
     segments: ?SegmentVector;
+    prepared: ?boolean
```

Comment:
With an eye towards https://github.com/mapbox/mapbox-gl-js/issues/5444, could we name this flag something specific to DEM tiles, so that it's easier to identify / pull out later?  E.g. `needsHillshadePrepare` or something